components/exercises/chapter-4-oscar/Utils.test.tsx

```diff
@@ -30,3 +35,40 @@ describe('arraySum', () => {
     expect(arraySum([-1, -2, -3, -4, -5])).toEqual(-15);
   });
 });
+
+describe('getReverseArray', () => {
+  it('should return an empty array if given an empty array', () => {
+    expect(getReverseArray([])).toEqual([]);
+  });
+
+  it('should return an array with the elements in reverse order', () => {
+    expect(getReverseArray([1, 2, 3, 4, 5])).toEqual([5, 4, 3, 2, 1]);
+    expect(getReverseArray(['a', 'b', 'c', 'd', 'e'])).toEqual([
+      'e',
+      'd',
+      'c',
+      'b',
+      'a',
+    ]);
+  });
+});
+
+describe('getReverseArrayInPlace', () => {
+  it('should return an empty array if given an empty array', () => {
+    const arr: any[] = [];
+    getReverseArrayInPlace(arr);
+    expect(arr).toEqual([]);
+  });
+
+  it('should return an array with the elements in reverse order', () => {
```

Comment:
can we add a test for an array with length different than 5?

Reply:
refactor adding length of 3 and length of 10